infra/gcp/clusters/modules/gke-cluster/versions.tf

```
@@ -15,7 +15,7 @@
  */
 
 terraform {
-  required_version = ">= 0.13"
+  required_version = ">= 0.13, < 0.14"
```

```suggestion
  required_version = "~> 0.13.0"
```
Ref https://www.terraform.io/docs/language/expressions/version-constraints.html

---

ping @ameukam to fix this (across this and other files) when you find time

---

@spiffxp fix applied in the PR.